ga.py

```diff
@@ -33,8 +40,28 @@ def assigned_priorities(prios, phenotype):
     return np.array(assigned_prios)
 
 
-def compute_costs(prios, phenotype):
-    return np.sum(assigned_priorities(prios, phenotype))
+def punish_erroneously_assigned_talkers(talkers, phenotype):
+    cost = [] # cost for talks where talker is not present in talks
+    for talker, talk in talkers.items():
+        c = 0
+        #punish if their first prio is not in the same slot as their talk
+        if talk in phenotype[0][0]:
```

Comment:
The second `[0]` is not necessary any more in the most recent version. Please rebase on the `main` branch.